x-pack/plugins/index_lifecycle_management/public/extend_index_management/components/add_lifecycle_confirm_modal.js

```diff
@@ -0,0 +1,282 @@
+    );
+  };
+  renderForm() {
+    const { policies, selectedPolicyName } = this.state;
+    const selectedPolicy = selectedPolicyName
+      ? policies.find(policy => policy.name === selectedPolicyName)
+      : null;
+
+    const options = policies.map(({ name }) => {
+      return {
+        value: name,
+        text: name,
+      };
+    });
+    options.unshift({
+      value: '',
+      text: i18n.translate(
+        'xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.choosePolicyMessage',
+        {
+          defaultMessage: 'Choose a lifecycle policy',
+        }
+      ),
+    });
+    return (
+      <EuiForm>
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.choosePolicyLabel"
+              defaultMessage="Lifecycle policy"
+            />
+          }
+        >
+          <EuiSelect
+            options={options}
+            value={selectedPolicyName}
+            onChange={e => {
+              this.setState({ selectedPolicyName: e.target.value });
+            }}
+            aria-label="Use aria labels when no actual label is in use"
+          />
+        </EuiFormRow>
+        {this.renderAliasFormElement(selectedPolicy)}
+      </EuiForm>
+    );
+  }
+  async componentDidMount() {
+    try {
+      const policies = await loadPolicies(false, this.props.httpClient);
+      this.setState({ policies });
+    } catch (err) {
+      showApiError(
+        err,
+        i18n.translate(
+          'xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.loadPolicyError',
+          {
+            defaultMessage: 'Error loading policy list',
+          }
+        )
+      );
+      this.props.closeModal();
+    }
+  }
+  render() {
+    const { policies } = this.state;
+    const { indexName, closeModal } = this.props;
+    const title = (
+      <FormattedMessage
+        id="xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.modalTitle"
+        defaultMessage="Add lifecycle policy to {indexName}"
+        values={{
+          indexName,
+        }}
+      />
+    );
+    if (!policies.length) {
+      return (
+        <EuiOverlayMask>
+          <EuiModal
+            onClose={closeModal}
+          >
+            <EuiModalHeader>
+              <EuiModalHeaderTitle >
+                {title}
+              </EuiModalHeaderTitle >
+
+            </EuiModalHeader>
+            <EuiModalBody>
+              <EuiCallOut
+                style={{ maxWidth: 400 }}
+                title={
+                  <FormattedMessage
+                    id="xpack.indexLifecycleMgmt.indexManagementTable.addLifecyclePolicyConfirmModal.noPoliciesWarningTitle"
+                    defaultMessage="No index lifecycle policies defined"
+                  />
+                }
+                color="warning"
+              >
+                <FormattedMessage
```

Comment:
this basically duplicates the title: 

![image](https://user-images.githubusercontent.com/1965714/48869480-3b422980-ed92-11e8-8e48-e0154bd9030d.png)

recommend to just have one line of text with a link like:
`No index lifecycle policies found. Click to define a lifecycle policy.`